docs/installation.md

```diff
@@ -26,7 +26,23 @@ git clone -b <latest-release-tag> https://github.com/cosmos/gaia
 cd gaia && make install
 ```
 
-> _NOTE_: If you have issues at this step, please check that you have the latest stable version of GO installed.
+If this command fails due to the following error message, you might have already set `LDFLAGS` prior to running this step.
+
+```
+# github.com/cosmos/gaia/cmd/gaiad
+flag provided but not defined: -L/usr/local/opt/foo
```

Comment:
Where does `/usr/local/opt/foo` come from?

Reply:
My exact error message had to do with compiling cryptography related packages for python, so mine says `-L/usr/local/opt/openssl/lib`, and I didn't want to put it verbatim and confuse readers.

I'll just leave that part out, and use this instead.

```
# github.com/cosmos/gaia/cmd/gaiad
flag provided but not defined: -L
...
make: *** [install] Error 2
```